src/Controls/src/Core/FlyoutPage/FlyoutPage.cs

```diff
@@ -68,9 +66,15 @@ public Page Detail
 					previousDetail?.SendDisappearing();
 					_detail?.SendAppearing();
 				}
-
-				previousDetail?.SendNavigatedFrom(new NavigatedFromEventArgs(_detail, NavigationType.PageSwap));
-				_detail?.SendNavigatedTo(new NavigatedToEventArgs(previousDetail));
+				
+				var destinationPage = _detail is NavigationPage currentNavPage ? currentNavPage.CurrentPage : _detail;
```

Comment:
I don't 'think we should change th ordering here

SendNavigating should fire before any other events. 
In theory at some point we will add cancellation args into "NavigatingFromEventArgs" that the user can cancel the navigation on

TabbedPage it fires Navigating First still

<img width="966" height="584" alt="image" src="https://github.com/user-attachments/assets/f5415f5b-00ef-42f2-8ba2-24dcef9a89a9" />

same with all the navigation bits

<img width="952" height="586" alt="image" src="https://github.com/user-attachments/assets/779c0bf5-f764-4297-aaa6-6008d2fc53dd" />